q2_diversity/_alpha/alpha_correlation_assets/src/render.js

```diff
@@ -17,7 +17,8 @@ export function render(svg, data) {
 
   const { xAxisLabel, yAxisLabel, minX, maxX, minY, maxY } = data;
 
-  const x = scaleLinear().domain([minX, maxX]).range([0, width]).nice();
+  // Add a ~3% padding to minX values, also handle for a minX of 0
+  const x = scaleLinear().domain([minX * 0.97 || minX - 0.03, maxX]).range([0, width]).nice();
```

Comment:
The `minX - 0.03` part isn't actually a percentage, so this produces some weird results for some metadata categories. How about:

```javascript
const x = scaleLinear().domain([minX - ((maxX - minX) * 0.03), maxX]).range([0, width]).nice();
```